scripts/parse_data_from_cldr.py

```diff
@@ -0,0 +1,151 @@
+import os
+import json
+from pprint import pprint
+import re
+
+SOURCE_PATH = "../raw_cldr_translation_data/"
+TARGET_PATH = "../number_parser/translation_data_merged"
+SUPPLEMENTARY_PATH = "../supplementary_translation_data/"
+
+REQUIRED_KEYS = ["spellout-cardinal", "spellout-numbering"]
+CAPTURE_BRACKET_CONTENT = r'\{(.*?)\}'
+
+LARGE_EXCEPTIONS = {}
+
+UNIT_NUMBERS = {}
+BASE_NUMBERS = {}
+MTENS = {}
+MHUNDREDS = {}
+MULTIPLIERS = {}
+VALID_TOKENS = {}
+
+def is_valid(key):
+    for each in REQUIRED_KEYS:
```

Comment:
I would rename `REQUIRED_KEYS` as `VALID_KEYS`.

On the other hand, `each` doesn't have any meaning, so I would rename it as `valid_key`:

```python
for valid_key in VALID_KEYS:
    if valid_key in key:
        return True
return False
```